packages/react-dom/src/events/isTextInputElement.js

```diff
@@ -28,8 +28,8 @@ const supportedInputTypes: {[key: string]: true | void, ...} = {
   week: true,
 };
 
-function isTextInputElement(elem: ?HTMLElement): boolean {
-  const nodeName = elem && elem.nodeName && elem.nodeName.toLowerCase();
```

Comment:
`elem.nodeName` truthy check was only needed when we passed in a `Window`. But it assumed that `window.nodeName` is falsy which breaks either down when using global variables or e.g. `<div id="nodeName" />`.

Reply:
I get this bug and find